### DISCUSSION TITLE
Papers in bivalves where DNA methylation AND gene expression are examined

### DISCUSSION THREAD

**Discussion by zbengt** [sr320/ceabigr#55] [General]
Compilation of papers where methylation and gene expression are examined started [here](https://docs.google.com/document/d/1vOVviAsp2d11_agjZap4fZKtelpMDnHbTcwO-38MEQ8/edit?usp=sharing). Feel free to add/comment things I've missed or point me in the right direction.

**Comment by sr320** [comment]
@zbengt check https://www.biorxiv.org/content/10.1101/2022.03.07.483338v1 - I think you might have missed a few.. and can you provide a summary of compilation?

**Reply by sr320** [reply]
Also it would be great to get these in a Paperpile Folder

**Reply by zbengt** [reply]
Sure thing! Sorry I posted this and didn't finish up. I'll get on it.
